_data/simple-icons.json

```
@@ -10837,6 +10837,11 @@
             "hex": "E40000",
             "source": "https://www.qantas.com"
         },
+        {
+            "title": "Qase",
+            "hex": "4F46DC",
+            "source": "https://qase.io"
```

```suggestion
            "source": "https://qase.io",
            "license": {
                "type": "custom",
                "url": "https://github.com/simple-icons/simple-icons/pull/9907#issuecomment-1831377594"
            }
```
Please add this to indicate special permission granted for use of Qase icon in simple-icons